pkg/satellite/satellitedb/projectmembers_test.go

```diff
@@ -163,3 +218,20 @@ func prepareUsersAndProjects(ctx context.Context, t *testing.T, users satellite.
 
 	return usersList, projectList
 }
+
+func TestSanitizedOrderColumnName(t *testing.T) {
```

Comment:
This really doesn't test anything. Maybe do this test with calling GetByProjectID.